.github/workflows/build.yml

```diff
@@ -167,15 +231,12 @@ jobs:
       run: cmake --build . --target mixxx-benchmark
       working-directory: cmake_build
       env:
-        QT_QPA_PLATFORM_PLUGIN_PATH: ${{ steps.buildenv.outputs.qt_qpa_platform_plugin_path }}
         # Render analyzer waveform tests to an offscreen buffer
-        QT_QPA_PLATFORM: offscreen
+        QT_QPA_PLATFORM: ${{ matrix.qt_qpa_platform }}
 
     - name: "Package"
       run: cpack -G ${{ matrix.cpack_generator }}
       working-directory: cmake_build
-      env:
-        PATH: ${{ steps.buildenv.outputs.path }}
```

Comment:
Bring this back. It ensures that `cpack` on macOS is the binary from the build environment rather than GitHub Actions.

Reply:
Please remember to always add comments for the purpose of everything that is not immediately clear.

Reply:
> Bring this back. It ensures that `cpack` on macOS is the binary from the build environment rather than GitHub Actions.

It's not needed. Instead of using `::set-output` in the buildenv script and then referencing that output everywhere, the script now appends a line `PATH=...` to the `$GITHUB_ENV` file. That way the variable persists between steps and doesn't need to be specified explicitly.